.github/workflows/Tests.yml

```diff
@@ -0,0 +1,42 @@
+name: "Run Tests"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref_name != github.event.repository.default_branch || github.ref != 'refs/tags/v*' }}
+
+jobs:
+  tests:
+    name: "Tests"
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1'
+        arch:
+          - x64
+          - x86
+        group:
+          - Interface
+          - Integrators
+          - Regression
+        include:
+          - coverage: false
+          - version: '1'
+            arch: x64
+            coverage: true
+        exclude: # test 32bit only with Julia 1 but not LTS
+          - version: '1.6'
+            arch: x86
```

Comment:
We don't test on Julia 1.6 anymore anyways it seems? So either 1.6 should be added above or we can remove this block completely.

Reply:
I wasn't sure if we wanted to test it with 1.6 or not, so I just left it in there to maintain status quo. I'll remove the block, thanks for confirming.

Reply:
Removed in 8770a61.